benchmarks/mwt/setup/01-cassandra-install/02-install-cassandra.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kudo.dev/v1beta1
```

Comment:
Any specific reason why we're not using the kudo cli here? In the other kuttl tests I've usually used something like

```
apiVersion: kudo.dev/v1beta1
kind: TestStep
commands:
  - command: kubectl kudo install --instance cassandra cassandra  -P params.yaml
    namespaced: true
```

and an extra `params.yaml` for the actual parameters. Makes it a lot easier for structured parameters as "NODE_TOPOLOGY" which is a yaml structure.

Reply:
no reason... I prefer to work within kuttl as much as possible and not shelling out with commands... but both ways are supported.